### ISSUE TITLE
0.9.0 does not work

### ISSUE THREAD

**Issue by mydea** [machty/ember-concurrency#285]
I tried to install 0.9.0 in an addon, and got the error that it couldn't import `ember-compatibility-helpers`. Manually installing that addon didn't help either. Downgrading to 0.8.27 made it work.

**Comment by stephen-flynn-fxpress** [comment]
I got the same when upgrading from 0.8.27 -> 0.9.0 (ember 3.8.0). 

In my case it seemed somehow to be caused by the transitive dependency on ember-concurrency/0.8.27 in ember-power-select/2.2.3. When I upgraded to ember-power-select/2.3.0, which depends on ember-concurrency/0.9.0, the problem went away. 

My error trace...

loader.js:247 Uncaught Error: Could not find module `ember-compatibility-helpers` imported from `ember-concurrency/-task-property`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
missingModule @ loader.js:247
findModule @ loader.js:258
Module.findDeps @ loader.js:168
findModule @ loader.js:262
Module.findDeps @ loader.js:168
findModule @ loader.js:262
Module.findDeps @ loader.js:168
findModule @ loader.js:262
Module.findDeps @ loader.js:168
findModule @ loader.js:262
requireModule @ loader.js:24
resolveInitializer @ index.js:11
registerInitializers @ index.js:28
_default @ index.js:65
(anonymous) @ app.js:97
Module.exports @ loader.js:106
requireModule @ loader.js:27
(anonymous) @ app-boot.js:3

**Comment by danwenzel** [comment]
@machty - Any ideas on this?  We keep encountering it as well.  We're having to add a resolution to our `package.json`:

```
"resolutions": {
  "ember-concurrency": "^0.8.27"
}
```

**Comment by allthesignals** [comment]
Seeing this too... looks like there was a minor version bump somewhere? I didn't change any dependencies, but yarn.lock shows some upgrades. Anyone know what's happening?

**Comment by jasonmevans** [comment]
We are encountering this as well +1

**Comment by donaldwasserman** [comment]
For what it's worth, this error is happening with `0.10.0` (In my case with `ec-decorators` `0.6.0`.

**Comment by cah-danmonroe** [comment]
I was having the same issue.  Resolved by upgrading e-concurrency and e-power-select to latest versions; 1.0.0 for EC and 2.3.3 for EPS

**Comment by mydea** [comment]
In an addon, it ended up being what was described here:
https://github.com/pzuraq/ember-compatibility-helpers/issues/32

TLDR, the addon had this in the index.js included hook:

```js
this._super.included(app)
```

Which should actually be:

```js
this._super.included.apply(this, arguments);
```

Pretty counter intuitive that this would lead to this error... But changing this made the addon run for me again (with 1.0.0).

**Comment by danwenzel** [comment]
For us, this turned out to be an issue with ember-power-select.  See https://github.com/cibernox/ember-power-select/issues/1232

**Comment by maxfierke** [comment]
Going to close this out.

To summarize the thread: please ensure nothing is depending upon an older version of ember-concurrency
